usr/bin/cylc

```diff
@@ -16,54 +16,132 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-# Wrapper for selection between installed versions of cylc.
-# Author: Matt Shin (Met Office).
-
-#_____________________________
-# INSTALLATION: 
-#
-#  1. Install this script as "cylc" in $PATH for normal users.
+#------------------------------------------------------------------------------
+# Wrapper script to support multiple Cylc versions installed on the same host.
+# Handles Cylc 8 Conda and Python virtual envs, and plain Cylc 7 installations.
```

Comment:
Cylc 8 could be done as plain installation couldn't it?
Also, cylc 7 could be installed using Conda or venv.
So, I suggest: "Handles Conda and Python virtual envs, and plain installations"

Reply:
> Cylc 8 could be done as plain installation couldn't it?


I guess, however, Cylc8 doesn't have the same `bin/cylc` pattern as Cylc7 so this would require extra logic. `CYLC_HOME` would have to point to the installed `cylc` script?

Reply:
OK - I'm happy that plain installations only support cylc 7.
"Handles Conda and Python virtual envs, and plain (Cylc 7) installations"

Reply:
A non-venv Cylc 8 seems a bit perverse as you'd have to install all the dependencies yourself. If anyone needs to do that, we can tweak the wrapper in a future PR.

Reply:
Also we can adjust later for Cylc 7 via conda or venv if necessary. That is difficult to do because of the pygtk dependencies - see #3958